src/Jackalope/Transport/DoctrineDBAL/Client.php

```diff
@@ -1146,6 +1133,8 @@ protected function xmlToColumns($xml, $propertyNames)
      * @param stdClass $data
      *
      * @throws InvalidArgumentException
+     *
+     * @deprecated This method was replaced with the `XmlToPropsParser` for performance reasons.
```

Comment:
This method is actually not longer called and if somebody did override it will have no effect, should we remove it. Would be a little bc break but actually don't think somebody used this method.

Reply:
hm. if somebody extended the client and called this method, then it would be ok to keep it as you did now.

if somebody extended the method, their changed method will no longer be called. that is already a BC break. though i agree with you it is unlikely anyone is using this.

i suggest keeping it as you now did it, with the deprecation annotation.

Reply:
I did also add a `@trigger` so it would also be logged.